### DISCUSSION TITLE
Project structure

### DISCUSSION THREAD

**Discussion by RechieKho** [mediagis/nominatim-docker#529] [Q&A]
I'm just wondering, why not utilise `git`'s feature such as branches and tags to manage version and instead copy pasting into different folders? The whole point of `git` version control is kind of to solve this issue...? I am very interested in the rationales behind this from the maintainers.

**Comment by leonardehrenfried** [comment]
I'm a maintainer and I don't really know myself. It had been that way when I joined the project.

We always support the two latest version and having separate folders makes this a bit easier.

**Comment by philipkozeny** [comment]
> I'm a maintainer and I don't really know myself. It had been that way when I joined the project.
> 
> We always support the two latest version and having separate folders makes this a bit easier.

We have inherited the structure from the original maintainer and have continued to utilize it. Might be a good idea to clean it up in the future.

**Comment by RechieKho** [comment]
Maybe you could create a `main` branch for cleaning up and utilising much conventional way of tagging release while leaving the old one in the `master` branch, saving it as a backup plan if it didn't work out, as changing a project structure is not an easy task and might cause regression.

**Reply by leonardehrenfried** [reply]
I have a request for you: please don't use loaded terms like "chaotic" and "ancient". It will make people not want to help you.

**Reply by RechieKho** [reply]
Very sorry, I shouldn't using such term. It was late night.
